src/main/resources/templates/progress/getHintList.html

```diff
@@ -28,9 +28,13 @@ <h2>Dostupné hinty pro Šifru <span th:text="${cypher.name}"></span></h2>
                 <td id="hintId" align="center" th:text="${hint.hintId}">value</td>
                 <td align="center" th:text="${hint.value}">value</td>
                 <td align="center">
-                    <button th:disabled="${takenHints.get(hint.hintId) != null}"
+                    <button th:if="${takenHints.get(hint.hintId) == null}"
                             th:onclick="'javascript:openDialog(' + ${hint.hintId} + ')'">Vzít Hint
                     </button>
+                    <a href="#" th:if="${takenHints.get(hint.hintId) != null}"
```

Comment:
Zobrazí se na tomhle tlačítku taky nějaký dialog pro potvrzení akce?

Reply:
Zaroven by se to nemuselo posilat get request, ale neco vhodnejsiho. V tomto pripade bych asi volil i DELETE, vzhledem k tomu, ze to maze z tabulky hint_taken. (Optional)

Reply:
V případě RESTu bych na 100% souhlasil, ale jakožto HTML noob se zeptám, jak by se tohle implementovalo tady? Použít form místo obyčejného <a> pro to tlačítko? Tam jde asi taky jen zadat jne POSTnebo GET.

Reply:
Upraveno na DELETE request.